src/multi_vector_simulator/utils/constants.py

```diff
@@ -265,6 +265,12 @@
         WARNING_TEXT: "allows defining the renewable share of the DSO supply (Values: Float). ",
         REQUIRED_IN_CSV_ELEMENTS: [ENERGY_PROVIDERS],
     },
+    EMISSION_FACTOR: {
+        DEFAULT_VALUE: 0,
```

Comment:
Aha! A default value to be cited in the `mvs_parameters.csv`.
You should also define `max_emissions`  here when time comes.

Reply:
> Aha! A default value to be cited in the `mvs_parameters.csv`.

see #713

> You should also define `max_emissions` here when time comes.

:+1: